README.md

```diff
@@ -168,90 +164,31 @@ To decide if your company really does fit our thesis and we should pursue an inv
+##### 1. Is _this_ the future?
+###### 1.1 Is this the future?
+###### 1.2 Is this vision massively ambitious?
+###### 1.3 What could this become?
+Suspend disbelief. If this _did_ work, what could it become? (eg: Air mattresses -> Airbnb)
 
-##### 2. Are they starting with an important problem? (The Problem/Market Fit Test)
+###### 1.4 Is this 10x better than the present?
 
+###### 1.5 Do people want this?
 _[42% of startups die because there’s no market need.](https://www.cbinsights.com/research/startup-failure-reasons-top/)_
-- Who desperately needs the product? (best: founder, second best: founders know the target user _perfectly_)
-- Is this a Top 3 problem for this target market?
-- Is it frequent and painful?
-- Is it under-served?
-- Would customers immediately adopt a solution if it was provided it to them right now?
-
-##### 3. Will this be a big market in 10 years, and will they win it? (The Great Market Test)
-
-- Why will this be a big market in 10 years?
-- Are they starting with a large part of a small market?
-- If this worked, could a [$100M ARR](http://christophjanz.blogspot.com/2014/10/five-ways-to-build-100-million-business.html), $250M GMV, or 10M user company be built in this market?
-
-##### 4. Is their Unique Value Proposition _unique_? (The Unique Value Prop Test)
-- Is this truly unique, or derivative? It’s easier to do something new and hard than something derivative and easy.
-- Is this _new_ - 10x better or cheaper?
-
-##### 5. What could this become? (The Black Swan Test)
-
-If this _did_ work, what could it become?
-
-Airbnb started as [strangers sleeping on air beds in your livingroom](https://medium.com/@bchesky/7-rejections-7d894cbaa084). Uber started as town cars for the 1%. As a fund, [the more experimental you are, the better the results](https://twitter.com/BenedictEvans/status/1008039058113626112).
-
-##### 6. Why now and not before? (The Insight/Timing/Secret Test)
+Who desperately needs the product? (best: founder, second best: founders know the target user _perfectly_)
 
-What change (like Drucker’s Seven Sources of Innovation) has created this opportunity?
-- **An Unexpected Success or Failure**: Has something unexpected happened?
-- **An Incongruity**: Is there a gap between the way things are and they way they should be?
-	- Between the reality of an industry and assumptions about it?
-	- Between assumed and actual customer values?
-- **A Process Need**: Are steps in a process missing, painful, or removable?
-- **An Industry or Market Change**: Are regulations, technologies, or customer preferences changing?
-- **A Demographics Change**: Are populations, income, education, or age changing?
-- **A Change in Perception, Meaning, or Mood**: Has a widely held view of life changed?
-- **New Knowledge**: Will a new technology make something possible, easier, cheaper, or more powerful?
+###### 1.6 What is the unique insight/secret that enables this?
+Is this truly unique, or derivative?
 
```

Comment:
Is this unique insight going to give an edge to accelerate them towards their vision?